src/main/java/edu/harvard/iq/dataverse/DatasetFieldType.java

```diff
@@ -164,8 +163,23 @@ public void setDisplayFormat(String displayFormat) {
         this.displayFormat = displayFormat;
     }
     
+    public Boolean isSanitizeHtml(){
+        if (this.fieldType.equals(FieldType.URL)){
+            return true;
+        }
+        return this.fieldType.equals(FieldType.TEXTBOX);
+    }
+    
+    public Boolean isEscapeOutputText(){
+        if (this.fieldType.equals(FieldType.URL)){
+            return false;
+        }
+        if (this.fieldType.equals(FieldType.TEXTBOX)){
+            return false;
+        }
+        return !(this.fieldType.equals(FieldType.TEXT) &&  this.displayFormat != null &&this.displayFormat.contains("<a"));
```

Comment:
Note: Check for uppercase `<A` as well as lowercase `<a`

(In the long run, debugging may be easier to make this a new type of field rather than having complex display logic)